lib/public/app.css

```diff
@@ -81,6 +81,10 @@ html, body {
     width: unset;
 }
 
+.mw-large {
+    max-width: var(--ui-component-large);
+}
+
```

Comment:
If you look at line 52, you'll see that:

- fixed width are put before percentage width, I would then put this line at l. 74
- we use `l` not `large`

Reply:
applied